Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
+
+# Clone an extra copy of the repo, to mess with during the tests
+RUN git clone https://github.com/release-depot/git_wrapper /tests/git_wrapper
+
+RUN git config --global user.name "GitWrapper Integration"
+RUN git config --global user.email "gitwrapper@example.com"
```

Comment:
We should be able to combine all 3 of the run lines into one. This will only create one container layer for this part of the build and should help over all build/run time.

Reply:
Thanks, I didn't realise that! Will update.